src/SFA.DAS.AAN.Hub.Data/Repositories/MemberDataCleanupRepository.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.AAN.Hub.Data.Entities;
+
+namespace SFA.DAS.AAN.Hub.Data.Repositories;
+
+public interface IMemberDataCleanupRepository
+{
+    Task<List<Member>> GetWithdrawnOrDeletedMembers();
+    Task<List<Member>> GetRemovedMembers();
+    Task UpdateMemberDetails(Member member, CancellationToken cancellationToken);
+    void DeleteMemberProfile(List<MemberProfile> memberProfiles, CancellationToken cancellationToken);
+    void DeleteMemberPreference(List<MemberPreference> memberPreferences, CancellationToken cancellationToken);
+    void DeleteMemberNotifications(List<Notification> memberNotifications, CancellationToken cancellationToken);
+    void DeleteMemberAudit(List<Audit> memberAudits, CancellationToken cancellationToken);
+    Task DeleteMemberApprentice(Member member, CancellationToken cancellationToken);
+    Task DeleteMemberEmployer(Member member, CancellationToken cancellationToken);
+    Task UpdateMemberFutureAttendance(Member member, CancellationToken cancellationToken);
+}
+
+[ExcludeFromCodeCoverage]
+public class MemberDataCleanupRepository : IMemberDataCleanupRepository
+{
+    private readonly IAanDataContext _context;
+
+    public MemberDataCleanupRepository(IAanDataContext context)
+    {
+        _context = context;
+    }
+    public async Task<List<Member>> GetWithdrawnOrDeletedMembers()
+    {
+        var statuses = new String[] { "Withdrawn", "Deleted" };
```

Comment:
Can you put withdrawn and deleted into constants, and use that. You can have "Removed" in the same set of constants (mentioned below)

Reply:
Or create enum which will be better